### PR TITLE
Schedule ui

### PR DESCRIPTION
**Pull request by eshiafr2**
Updating event Info xml page.

### CODE REVIEW

File: app/src/main/res/layout/schedule_event_list_item.xml [hunk 1]
```diff
@@ -4,114 +4,114 @@
     xmlns:tools="http://schemas.android.com/tools"
```
Comment (patrickfeltes) [inline app/src/main/res/layout/schedule_event_list_item.xml]: Can you delete this file and use `event_tile.xml` instead? I had to create an xml for the homepage tiles, so we should just use the same one across the app. You'll have to change `EventsAdapter.kt` to use the event_tile.xml and then update the view ids in that class.

File: app/src/main/java/org/hackillinois/android/view/EventInfoActivity.kt [hunk 2]
```diff
@@ -52,13 +52,13 @@ class EventInfoActivity : AppCompatActivity() {
...
             eventDescription.text = it.description
 
-            if (it.locations.isEmpty()) {
-                directionsButton.visibility = View.GONE
-            } else {
-                directionsButton.visibility = View.VISIBLE
-                val location = LatLng(it.locations[0].latitude, it.locations[0].longitude)
-                directionsButton.setOnClickListener(DirectionsOnClickListener(location, event.name))
-            }
+//            if (it.locations.isEmpty()) {
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/EventInfoActivity.kt]: Don't commit commented code!

Comment (patrickfeltes) [reply app/src/main/java/org/hackillinois/android/view/EventInfoActivity.kt]: This was me :(

Comment (bstrauch24) [reply app/src/main/java/org/hackillinois/android/view/EventInfoActivity.kt]: :(

File: app/src/main/java/org/hackillinois/android/view/schedule/EventsAdapter.kt [hunk 3]
```diff
@@ -7,7 +7,11 @@ import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import kotlinx.android.synthetic.main.schedule_event_list_item.view.*
+import kotlinx.android.synthetic.main.activity_event_info.view.*
+import kotlinx.android.synthetic.main.activity_event_info.view.eventLocation
+import kotlinx.android.synthetic.main.activity_event_info.view.eventTitle
+import kotlinx.android.synthetic.main.event_tile.view.*
+//import kotlinx.android.synthetic.main.schedule_event_list_item.view.*
```
Comment (patrickfeltes) [inline app/src/main/java/org/hackillinois/android/view/schedule/EventsAdapter.kt]: Can you delete this line?

Comment (eshiafr2) [reply app/src/main/java/org/hackillinois/android/view/schedule/EventsAdapter.kt]: Oh sorry I forgot we shouldn't commit commented code. Thanks.

**Pull request merged**